### ISSUE TITLE
Support for PHP 8 and Css-selector 6

### ISSUE THREAD

**Issue by andreinocenti** [FriendsOfPHP/Goutte#453]
When trying to add it via Composer i'm having the following issue:

```
 Problem 1
    - fabpot/goutte v4.0.0 requires php ^7.1.3 -> your php version (8.0.11) does not satisfy that requirement.
    - fabpot/goutte v4.0.1 requires symfony/css-selector ^4.4|^5.0 -> found symfony/css-selector[v4.4.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires fabpot/goutte ^4.0 -> satisfiable by fabpot/goutte[v4.0.0, v4.0.1].
```

Looks it doesnt support php 8 neither css-selector 6

Thank you

**Comment by nathanaelmartel** [comment]
same problem with symfony/browser-kit v6.0.1 : 
```  Problem 1
    - fabpot/goutte v4.0.0 requires php ^7.1.3 -> your php version (8.0.11) does not satisfy that requirement.
    - fabpot/goutte v4.0.1 requires symfony/browser-kit ^4.4|^5.0 -> found symfony/browser-kit[v4.4.0, ..., v4.4.27, v5.0.0, ..., v5.4.0] but the package is fixed to v6.0.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires fabpot/goutte ^4.0 -> satisfiable by fabpot/goutte[v4.0.0, v4.0.1].
```

(I used it with symfony 5.4 + php8 and it was working well, so I guess the problem is not php8, but symfony 6 dependancies)

**Comment by fabpot** [comment]
Fixed in the latest version
